docs/resources.md

```diff
@@ -0,0 +1,39 @@
+# Resources
+
+The following are resources about Melange collected around the web.
+
+## Blog posts, podcasts, talks
+
+| Date | Type | Description |
+|----- | ---- | ----------- |
+| Sep 2024 | talk | [Type-Safe React with OCaml by Dillon Mulroy](https://reactalicante.es/#schedule) (video not yet uploaded) |
+| Sep 2024 | podcast | [The State of Full-Stack OCaml (with António Monteiro)](https://pod.link/developer-voices/episode/4666dc4d7aa59599852c02b6415a36d4) ([video](https://www.youtube.com/watch?v=vSSpbAI_-t0))
+| May 2024 | blog | [Melange 4.0 is here](https://melange.re/blog/posts/melange-4-is-here)
+| Apr 2024 | blog | [A Preview of Universal Libraries in Dune](https://melange.re/blog/posts/dune-universal-libraries-preview)
+| Apr 2024 | talk | [Dillon Mulroy — Melange: The Next Frontier in Type-Safe Web Development](https://www.youtube.com/watch?v=wl8zUq1FUzM)
+| Apr 2024 | blog | [What's 2024 brought to Melange so far?](https://melange.re/blog/posts/whats-2024-brought-to-melange-so-far)
+| Feb 2023 | blog | [Announcing Melange 3](https://melange.re/blog/posts/announcing-melange-3)
+| Nov 2023 | blog | [Building React Server Components in OCaml](https://tech.ahrefs.com/building-react-server-components-in-ocaml-81c276713f19) |
+| Oct 2023 | blog | [The rest of 2023 in Melange](https://melange.re/blog/posts/the-rest-of-2023-in-melange)
+| Sep 2023 | blog | [Beyond TypeScript: Differences Between Typed Languages](https://tech.ahrefs.com/beyond-typescript-differences-between-typed-languages-f3e14253) |
+| Sep 2023 | blog | [Introducing Melange 2.0](https://melange.re/blog/posts/introducing-melange-20) |
+| July 2023 | blog | [What's next for Melange](https://melange.re/blog/posts/whats-next-for-melange) |
+| May 2023 | podcast | [David Sancho chats to Antonio Monteiro about OCaml and Melange](https://www.youtube.com/watch?v=zKO-Y2_a-yk) |
```

Comment:
There's another talk with Antonio, a bit older https://watch.ocaml.org/w/aEabwHEfXTKGxTNi6Xhy2a

Reply:
Added in https://github.com/melange-re/melange-re.github.io/pull/189/commits/a137ca1b37653d8b39377878bf52f5908d2ac384